### PR TITLE
Added missing metric to list

### PR DESCRIPTION
**Pull request by pohannigansfx**
Added metric to listing and rebranded from SignalFx to Splunk Infrastructure Monitoring.

### CODE REVIEW

File: signalfx-org-metrics/metrics.yaml [hunk 1]
```diff
@@ -479,15 +479,17 @@ sf.org.num.gcpStackdriverClientCallCountThrottles:
...
 
-sf.org.num.metric:
-  brief: Number of unique metrics across all MTS
-  description: 'Number of unique metrics across all MTS.
+sf.org.num.LimitedMetricTimeSeriesCreateCallsByCategoryType:
+  brief: Number of metric time series created that are over the limit for an MTS category
+  description: 'Number of metric time series created that are over the purchased
+    limit for that MTS category (such as Host, Container, Custom, Tracing and Bundled).
 
     Dimension(s): `orgId`
```
Comment (tedoc2000) [inline signalfx-org-metrics/metrics.yaml:487]: Dimension(s): `orgId` and `categoryType`

File: signalfx-org-metrics/metrics.yaml [hunk 2]
```diff
@@ -479,15 +479,17 @@ sf.org.num.gcpStackdriverClientCallCountThrottles:
   metric_type: counter
   title: sf.org.num.gcpStackdriverClientCallCountThrottles
 
-sf.org.num.metric:
-  brief: Number of unique metrics across all MTS
-  description: 'Number of unique metrics across all MTS.
+sf.org.num.LimitedMetricTimeSeriesCreateCallsByCategoryType:
```
Comment (tedoc2000) [inline signalfx-org-metrics/metrics.yaml:482]: sf.org.numLimitedMetricTimeSeriesCreateCalls

File: signalfx-org-metrics/metrics.yaml [hunk 3]
```diff
@@ -479,15 +479,17 @@ sf.org.num.gcpStackdriverClientCallCountThrottles:
...
+  brief: Number of metric time series created that are over the limit for an MTS category
+  description: 'Number of metric time series created that are over the purchased
+    limit for that MTS category (such as Host, Container, Custom, Tracing and Bundled).
 
     Dimension(s): `orgId`
 
     Data Resolution: 15 minutes'
   metric_type: gauge
-  title: sf.org.num.metric
+  title: sf.org.num.LimitedMetricTimeSeriesCreateCallsByCategoryType
```
Comment (tedoc2000) [inline signalfx-org-metrics/metrics.yaml:491]: sf.org.numLimitedMetricTimeSeriesCreateCalls

File: signalfx-org-metrics/metrics.yaml [hunk 4]
```diff
@@ -479,15 +479,17 @@ sf.org.num.gcpStackdriverClientCallCountThrottles:
...
+sf.org.num.LimitedMetricTimeSeriesCreateCallsByCategoryType:
+  brief: Number of metric time series created that are over the limit for an MTS category
+  description: 'Number of metric time series created that are over the purchased
+    limit for that MTS category (such as Host, Container, Custom, Tracing and Bundled).
 
     Dimension(s): `orgId`
 
     Data Resolution: 15 minutes'
   metric_type: gauge
-  title: sf.org.num.metric
```
Comment (tedoc2000) [inline signalfx-org-metrics/metrics.yaml:490]: Undo this remove?

**Review by tedoc2000 — CHANGES_REQUESTED** [review]

File: signalfx-org-metrics/README.md.jinja [hunk 5]
```diff
@@ -11,20 +11,20 @@
...
--  Metrics that tell you how SignalFx pulls data on your behalf from cloud services like Amazon Web Services or Google Cloud Platform, such as the number of calls to each GCP Stackdriver client method or how many of the calls to the AWS CloudWatch API are being throttled by AWS.
+-  Metrics that tell you how the application pulls data on your behalf from cloud services like Amazon Web Services or Google Cloud Platform, such as the number of calls to each GCP Stackdriver client method or how many of the calls to the AWS CloudWatch API are being throttled by AWS.
 
 Your organization is not charged for these metrics, and they do not count against any limits.
 
 
 ## USAGE
 
-SignalFx admins can see some of these values in built-in charts on the Organization Overview page. And, like any other metrics, you can display them on charts you build to monitor your system health.
+Application admins can see some of these values in built-in charts on the Organization Overview page. And, like any other metrics, you can display them on charts you build to monitor your system health.
```
Comment (rpille-sfx) [inline signalfx-org-metrics/README.md.jinja:27]: I suggest saying "Admins" instead of "Application admins" here.

Comment (pohannigansfx) [reply signalfx-org-metrics/README.md.jinja:27]: done

File: signalfx-org-metrics/metrics.yaml (hunk 4 above)
Comment (pohannigansfx) [reply signalfx-org-metrics/metrics.yaml:490]: That was title from another entry I'd copied over as a syntax model for the new metric.

File: signalfx-org-metrics/metrics.yaml (hunk 3 above)
Comment (pohannigansfx) [reply signalfx-org-metrics/metrics.yaml:491]: done

**Comment by pohannigansfx** [comment]
Made revisions suggested by two reviewers

**Pull request merged**